app.css

```diff
@@ -36,3 +36,37 @@ body {
 .head-wrapper header ul li:hover {
     border-bottom: 1px solid #4cc5e2;
 }
+
+/*Slick*/
+.single-item {
+  direction: ltr;
+  margin: auto;
+  margin-top:80px;
+  height:600px;
+  width: 960px;
+  border:1px solid black; /*will be removed later on*/
+}
+
+/*Icons menu bar*/
+.iconMenuBar {
+  margin-top: 80px;
+  height: 400px;
+  background-color: #373d46;
+  color: #4cc5e2;
+}
+.iconMenuBar ul {
+  text-align:center;
+}
+.iconMenuBar ul li {
+  padding: 100px 60px 12px 60px;
```

Comment:
This is bad practice and will result in a less responsive behaviour.
You should position these 3 elements using relative width (%)